test/functional/test-ad-cid.js

```diff
@@ -52,16 +52,16 @@ function tests(name) {
 
       afterEach(() => {
         sandbox.restore();
-        delete clientIdScope['with_cid'];
+        delete clientIdScope['_ping_'];
```

Comment:
ad `layoutCallback` needs `render-start` to resolve.  'with-cid' is an ad type that we don't recognize, and we won't generate `render-start` for it.  Thus getAd() will never succeed and we will get timeout for this test.  
I changed it to '_ping_' to solve this issue.